sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -222,6 +222,7 @@ documentSentiment.getSentences().forEach(sentenceSentiment ->
     System.out.printf("Analyzed sentence sentiment: %s.%n", sentenceSentiment.getSentiment()));
 ```
 For samples on using the production recommended option `AnalyzeSentimentBatch` see [here][analyze_sentiment_sample].
+For samples on sentiment analysis with opinion mining see [here][analyze_sentiment_with_opinion_mining_sample].
```

Comment:
We could make this stand out more by adding it to the description of Analyze Sentiment.
What about something like:
```
Run a Text Analytics predictive model to identify the positive, negative, neutral or mixed sentiment contained in the 
provided document or batch of documents.
To get more granular information about the opinions related to aspects of a product/service, also knows as Aspect-based Sentiment Analysis in Natural Language Processing (NLP), see sample on sentiment analysis with opinion mining see [here][analyze_sentiment_with_opinion_mining_sample].
```

Reply:
That works. I will update it.